packages/patternfly-4/react-core/src/components/Select/SelectOption.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import styles from '@patternfly/patternfly-next/components/Select/select.css';
+import { css } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+import { SelectContext } from './selectConstants';
+
+const propTypes = {
+  /** the value for the option */
+  children: PropTypes.string,
```

Comment:
I don't think we don't have a need for children here since we are using value.

Reply:
I left it in as an alternate declaration method. So users could still do <SelectOption>value</SelectOption> as well as <SelectOption value={value} />. Should I remove it?

Reply:
Think we should get rid of children for now.  If this comes in as a separate request we could discuss this option further.